consultation_analyser/support_console/jinja2/support_console/consultations/show.html

```diff
@@ -74,6 +74,37 @@ <h2 class="govuk-heading-m">Users</h2>
         Add users
       </a>
 
+      <h2 class="govuk-heading-m">Previous runs of the theme generation</h2>
+      <table class="govuk-table">
+        <thead class="govuk-table__head">
+          <tr class="govuk-table__row">
+            <th scope="col" class="govuk-table__header">Link</th>
+            <th scope="col" class="govuk-table__header">Run started at</th>
+            <th scope="col" class="govuk-table__header">Run finished at</th>
+          </tr>
+        </thead>
+        <tbody class="govuk-table__body">
+        {% for run in processing_runs %}
+          <tr class="govuk-table__row">
+            <td class="govuk-table__cell">
+              <a href="{{ url('consultation_run', kwargs={'consultation_slug': consultation.slug, 'processing_run_slug': run.slug}) }}" class="govuk-link govuk-body govuk-link--no-visited-state">
+                View on frontend
+              </a>
+            </td>
+            <td class="govuk-table__cell">
+              {% if run.started_at %}
+                {{ datetime(run.started_at) }}
+              {% endif %}
+            </td>
+            <td class="govuk-table__cell">
+              {% if run.started_at %}
+                {{ datetime(run.finished_at) }}
```

Comment:
If a run fails to complete this will crash — let's discuss how to handle?

Reply:
Doh, I think that's a lazy copy & paste error, should be `{% if run.finished_at %}` i.e. only show finished at if it exists. 

Though I don't know if there's anything else we need to consider if a run fails to complete?

Reply:
Have updated to be `{% if run.finished_at %}`.